### PR TITLE
[Doc] Fix bad links @open sesame 12/01 19:10

### PR DESCRIPTION
**Pull request by niley7464**
This patch fixes bad links in documentation.

Signed-off-by: yelini-jeong <yelini.jeong@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [X]Skipped
2. Run test: [ ]Passed [ ]Failed [X]Skipped

### CODE REVIEW

File: Documentation/tutorials/tutorial5_gstreamer_api.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: T5. GStreamer API
 The `gst-launch` tool is convenient, but it is recommended to use it simply to test pipeline description.  
 In this tutorial, let's learn how to create an application. (Full example source code can be found [here](https://github.com/nnstreamer/nnstreamer-example/blob/main/native/example_object_detection_tensorflow_lite/nnstreamer_example_object_detection_tflite.cc)).  
 Creating a pipeline application is simple. You can use `gst_parse_launch ()` which is GStreamer API instead of `gst-launch` tool.  
-Refer to [here]((Documentation/how-to-run-examples.md)) for building nnstreamer-example.
+Refer to [here](Documentation/how-to-run-examples.md) for building nnstreamer-example.
```
Comment (gichan-jang) [inline Documentation/tutorials/tutorial5_gstreamer_api.md:9]: This change will also fix the broken link in github.
```suggestion
Refer to [here](../how-to-run-examples.md) for building nnstreamer-example.
```

Comment (niley7464) [reply Documentation/tutorials/tutorial5_gstreamer_api.md]: Thank you! I canged it.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM 👍

**Review by again4you — APPROVED** [review]
LGTM!

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**